test/test_browser.py

```diff
@@ -761,6 +761,7 @@ def test_sdl_surface_lock_opts(self):
     # Test Emscripten-specific extensions to optimize SDL_LockSurface and SDL_UnlockSurface.
     self.btest('hello_world_sdl.cpp', reference='htmltest.png', args=['-DTEST_SDL_LOCK_OPTS', '-lSDL', '-lGL'])
 
+  @also_with_wasmfs
```

Comment:
Are there no tests other/core that can now pass with wasmfs (that couldn't before)?

Reply:
I didn't notice one. Perhaps all of our file preload tests are in `browser`?